Apollo.xcodeproj/project.pbxproj

```diff
@@ -4496,6 +4496,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		90690D322243442F00FC2E54 /* Ensure no build settings are in the Xcode project */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
Any further details on what this new attribute is for?

Reply:
It indicates that the build script running is NOT dependent on any input or output sources.

@calvincestari I know you know this part, but for the sake of a complete answer: If you provide any of these sources it will only run them when the input sources have changed or the output sources do not exist.

In past versions of Xcode, if you don't provide any input or output sources, it will just always run the script. In Xcode 14, not providing input sources will show a warning. I don't want to go find the actual warning message right now, but its something along the lines of "This script will run on every build of your target. Please provide input/output sources. If you want this script to run every time, check the checkbox for 'not dependent on any sources'"

Checking that checkbox on the script build phase will make `alwaysOutOfDate = 1;`.

Reply:
Maybe we want to setup an input source for that build phase to only run when the xcproject file changes actually though!

Reply:
> In past versions of Xcode, if you don't provide any input or output sources, it will just always run the script. In Xcode 14, not providing input sources will show a warning.

This is the key bit I didn't know about Xcode 14 - thanks.

Reply:
> Maybe we want to setup an input source for that build phase to only run when the xcproject file changes actually though!

It seems you need to specify both input _and_ output files for that setting to not generate any warnings. Apple's [documentation](https://developer.apple.com/documentation/Xcode/improving-the-speed-of-incremental-builds) says to generate a static file in the case of no output.

The execution time of that custom script is 0.013s? We _can_ generate a static file as 'output' but I think the implementation would take way longer than the eventual time you'd end up saving.

![Screen Shot 2022-08-15 at 3 10 35 PM](https://user-images.githubusercontent.com/146856/184641763-7bb0aa86-da30-4903-8716-af4b2726d3bb.png)